PF/Assignment1/assignment5.java

```diff
@@ -0,0 +1,62 @@
+package programing_fundamental;
+
+import java.util.*;
+
+public class assignment5 {
+	public int checkArraySort(int[] input)
+	{
+		int i,j,flag=0;
+		
+		if(input.length==1)
+			return 1;
+		for(i=0;i<input.length;i++)
```

Comment:
ACTION: This can be done by just iterating once instead of it two times iterating in nested loops. You can just compare first and second element of array if first element is  less than second element of array that means array is either ascending sorted or not sorted. then you can iterate on array and check weather it maintains ascending sort order by comparing two consecutive elements, If it doesn't maintain sort order directly return 0. same thing you can do it in else (descending order) case .